tests/test_bs.py

```diff
@@ -61,11 +62,22 @@ def setup_bootstrap_in_namespace(namespace, bs_deployment_info, bootstrap_config
     cspec.append_args(genesis_time=GENESIS_TIME.isoformat('T', 'seconds'),
                       **bootstrap_args)
 
-    resp = deployment.create_deployment(BOOT_DEPLOYMENT_FILE, namespace,
-                                        deployment_id=bs_deployment_info.deployment_id,
-                                        replica_size=bootstrap_config['replicas'],
-                                        container_specs=cspec,
-                                        time_out=dep_time_out)
+    dep_type = 'deployment' if 'deployment_type' not in bootstrap_config else bootstrap_config['deployment_type']
```

Comment:
so we need to add deployment_type to config.yaml ?

Reply:
only if you want to explicitly use statefulset (consistent storage)
I prfered to leave the option to use deployment since provisionning storgae takes an extra time